### PR TITLE
Introduce converters between touches, neurons to parquet and back

### PR DESCRIPTION
**Pull request by matz-e**
Built on top of #807.

### CODE REVIEW

File: bbp/hpc/highfive/default.nix [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
 stdenv.mkDerivation rec {
   name = "highfive-${version}";
-  version = "1.4-dev201710";
+  version = "1.4-dev201805";
 
   src = fetchFromGitHub {
     owner = "BlueBrain";
     repo = "HighFive";
-    rev = "fce248203ca88c412fb32d5cb862628352d0bfe5";
+    rev = "6be97a0efcc618f1033a3745ff4030dc28cdd167";
```
Comment (tristan0x) [inline bbp/hpc/highfive/default.nix]: Do you need to bump highfive?

File: bbp/hpc/syntool/default.nix [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
-    version = "0.1-dev201709";
+    version = "0.1-dev201805";
 
 
     src = fetchgitPrivate {
         url = "ssh://bbpcode.epfl.ch/hpc/synapse-tool";
-        rev = "f02fff48867b3acd9162515fdc224f1b8f48e9df";
-        sha256 = "15iarkk9fsama2mdj0ji5zcknf8r9z6490fcinznrn8fvv1lnd0z";
+        rev = "6ef3eb401f3fca5badfaa77d40ae6728970c6cba";
+        sha256 = "06iwz15f67nb01y904xwy9j52qk8bhflclkyi4b488k4qh91cakf";
```
Comment (tristan0x) [inline bbp/hpc/syntool/default.nix]: Do you need to bump syntool?

File: config/bbp_maintainers_config.nix [hunk 3]
```diff
@@ -8,6 +8,7 @@
     FrancescoCasalegno = "Casalegno Francesco <francesco.casalegno@epfl.ch>";
     jamesgkind = "James Gonzalo King <jamesgonzalo.king@epfl.ch>";
     jplanasc = "Judit Planas <judit.planas@epfl.ch>";
+    matze = "Matthias Wolf <matthias.wolf@epfl.ch>";
```
Comment (tristan0x) [inline config/bbp_maintainers_config.nix]: You should add yourself to the maintainers list of the recipe you added

File: bbp/hpc/converters/default.nix [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    cd neuron_parquet
+  ";
+
+  meta = {
+    description = "Convert touches and neurons between different formats";
+    platforms = stdenv.lib.platforms.unix;
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/building/Functionalizer";
+    repository = "ssh://bbpcode.epfl.ch/building/Functionalizer";
+    license = {
+      fullName = "Copyright 2012, Blue Brain Project";
```
Comment (tristan0x) [inline bbp/hpc/converters/default.nix]: copyright is outdated.

File: bbp/hpc/highfive/default.nix (hunk 1 above)
Comment (adevress) [reply bbp/hpc/highfive/default.nix]: We found together that the converter requires a recent HighFive on friday

File: bbp/hpc/syntool/default.nix (hunk 2 above)
Comment (matz-e) [reply bbp/hpc/syntool/default.nix]: The converters require MPI additions that Fernando added to that branch.

File: config/bbp_maintainers_config.nix (hunk 3 above)
Comment (matz-e) [reply config/bbp_maintainers_config.nix]: Done.

**Comment by tristan0x** [comment]
retest this please

**Comment by matz-e** [comment]
@tristan0x I renamed the recipe as discussed this morning. This includes the latest changes that I just finished testing/verifying and should be stable for a while.

**Comment by adevress** [comment]
add to whitelist

**Pull request merged**